includes/install.sh

```diff
@@ -1,27 +1,36 @@
+# shellcheck disable=SC2148,SC1091
+source /includes/colors.sh
+
+steamcmd_dir="/home/steam/steamcmd"
+
 # Function to install the gameserver
 function install_server() {
-    # force a fresh install of all
-    echo ">>> Doing a fresh install of the gameserver"
+    # Force a fresh install of all
+    ew ">>> Doing a fresh install of the gameserver...\n"
```

Comment:
I strongly dislike that i have to add to every echo an \n that adds no value, that should be in the alias function or even deeper.

Adds no value, adds complexitiy, adds code-dublication. Does nothing really.